custom_components/pentair_cloud/coordinator.py

```
@@ -71,3 +76,55 @@ async def _async_update_data(self):
+                    self.data,
+                    device,
+                    ignore_order=True,
+                    report_repetition=True,
+                    verbose_level=2,
+                )
+                _LOGGER.debug("Devices updated: %s", diff if diff else "no changes")
+                return device
+        except Exception as err:  # pylint: disable=broad-except
+            _LOGGER.error(
+                "Unknown exception while updating Pentair data: %s", err, exc_info=1
+            )
```

Ruff (0.15.0)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)